cvat/apps/engine/static/engine/js/shapeCreator.js

```diff
@@ -146,15 +147,15 @@ class ShapeCreatorController {
             let shortkeys = window.cvat.config.shortkeys;
 
             let switchDrawHandler = Logger.shortkeyLogDecorator(function() {
-                this.switchCreateMode(false);
+                this.switchCreateMode(false, true);
             }.bind(this));
 
             Mousetrap.bind(shortkeys["switch_draw_mode"].value, switchDrawHandler.bind(this), 'keydown');
         }
     }
 
-    switchCreateMode(force) {
-        this._model.switchCreateMode(force);
+    switchCreateMode(force, usingShortkey) {
```

Comment:
What about default value? This method is called with a single arguments in many places